README.md

```diff
@@ -0,0 +1,40 @@
+
+Hello world
+This is the project for day02/week01/module01 of the microverse program.
+
+Built With
+HTML
+CSS
+
+
+
+
+To get a local copy up and running follow these simple example steps.
+
+Prerequisites
+Setup
+Install
+Usage
+Run tests
+Deployment
```

Comment:
- [ ] [OPTIONAL] Removing the empty sections will make your README file more professional

Reply:
Thank you ! I will remove this part and do the merging.